src/PortalModal.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react';
+import ReactDOM from 'react-dom';
+
+interface PortalModalProps {
+    children: ReactNode,
+};
+
+const Modal = ({ 
+    children 
+}: PortalModalProps) => {
+    const el = document.getElementById('modal-root') as HTMLElement;
```

Comment:
중요한 건 아닌데, div엘리먼트라면 HTMLDivElement로 설정하는 것도 괜찮을 것 같아요.

```suggestion
    const el = document.getElementById('modal-root') as HTMLDivElement;
```